api/HTMLSourceElement.json

```diff
@@ -0,0 +1,298 @@
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox_android": {
+              "version_added": "4"
+            },
+            "opera_android": {
+              "version_added": true
+            },
+            "safari_ios": {
+              "version_added": true
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "sizes": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/sizes",
+          "support": {
+            "edge": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "33",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "dom.image.picture.enabled",
+                  "value_to_set": "true"
+                }
+              ]
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox_android": {
+              "version_added": "33",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "dom.image.picture.enabled",
+                  "value_to_set": "true"
+                }
+              ]
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "src": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLSourceElement/src",
+          "support": {
+            "chrome": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "3.5"
+            },
+            "ie": {
+              "version_added": "9"
+            },
+            "opera": {
+              "version_added": true
+            },
+            "safari": {
+              "version_added": true
+            },
+            "webview_android": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox_android": {
+              "version_added": "4"
+            },
+            "opera_android": {
+              "version_added": true
+            },
+            "safari_ios": {
+              "version_added": true
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "srcset": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLSourceElement/srcset",
+          "support": {
+            "edge": {
```

Comment:
Please add `webview_android`, `chrome`, and `chrome_android` with the value `true`.